docker-compose.yml

```diff
@@ -1,24 +0,0 @@
-version: "2.1"
```

Comment:
I think we still want to keep a generic `docker-compose.yml` right? When running additional setups, you cascade them with additional dockers like how they do with `docker-compose.override.yml`. I see the same potential here for the full service, using `docker-compose.yml` holding the base configurations. https://docs.docker.com/compose/extends/